### ISSUE TITLE
Certificates: Create an API endpoint to generate certificates

### ISSUE THREAD

**Issue by arslanashraf7** [mitodl/mitxonline#700]
As a Learner, I'd like to get my certificate generated in MITxOnline once I complete the course on edX

We need to create an API endpoint to generate certificates for users. Ideally, this endpoint should only need the `user_id`, `course_id` for generating the certificate. As an enhancement, we can make this API accept a list of users so that we can use this to generate the certificates in a  bulk, although the current requirement would be satisfied with a single user.

This API endpoint should be behind an authentication layer. We need to decide upon the access (Either any authenticated or just the user with Admin access)

This API should be added to the certificates app that will be created as a result of https://github.com/mitodl/mitxonline/issues/686

#### Dependencies:
This ticket depends upon https://github.com/mitodl/mitxonline/issues/691

#### Designs and Mockups
None

#### Acceptance Criteria:

- [ ] An API endpoint that generates the certificate per user per course run

**Comment by rhysyngsun** [comment]
See my comment on https://github.com/mitodl/mitxonline/issues/690#issuecomment-1188356895

**Comment by pdpinch** [comment]
I don't think we need this if we go with #717. I'm going to label this "On Hold"
